Dockerfile

```diff
@@ -16,19 +16,55 @@ RUN make build
 # Node
 FROM alpine:3.17 as kanvas-node
 COPY --from=builder /app/bin/kanvas-node /usr/local/bin
+
+RUN addgroup user && \
+    adduser -G user -s /bin/sh -h /home/user -D user && \
+    mkdir /kanvas_log/ && \
+    chown user:user /kanvas_log
+
+USER user
+WORKDIR /home/user/
+
 ENTRYPOINT ["kanvas-node"]
 
 # Stateviz
 FROM alpine:3.17 as kanvas-stateviz
 COPY --from=builder /app/bin/kanvas-stateviz /usr/local/bin
+
+RUN addgroup user && \
+    adduser -G user -s /bin/sh -h /home/user -D user && \
+    mkdir /kanvas_log/ && \
+    chown user:user /kanvas_log
+
+USER user
+WORKDIR /home/user/
+
 CMD ["kanvas-stateviz"]
 
 # Batcher
 FROM alpine:3.17 as kanvas-batcher
 COPY --from=builder /app/bin/kanvas-batcher /usr/local/bin
+
+RUN addgroup user && \
+    adduser -G user -s /bin/sh -h /home/user -D user && \
+    mkdir /kanvas_log/ && \
+    chown user:user /kanvas_log
```

Comment:
FYI, batcher and validator do not required a `kanvas_log` file.

Reply:
Oh thanks i'll remove the lines on `batcher` and `validator`